src/SFA.DAS.LevyTransferMatching/Application/Queries/Pledges/GetAmount/GetAmountQuery.cs

```diff
@@ -5,5 +5,7 @@ namespace SFA.DAS.LevyTransferMatching.Application.Queries.Pledges.GetAmount
     public class GetAmountQuery : IRequest<GetAmountQueryResult>
     {
         public string EncodedAccountId { get; set; }
+
+        public long AccountId { get; set; }
```

Comment:
I suspect that this value replaces `EncodedAccountId`, in which case the latter should be removed.

Reply:
You mean AccountId should be removed?  This is still referenced at 6 places?

Reply:
No, I meant `EncodedAccountId` should be removed, assuming it is no longer referenced?

Reply:
This needs some further investigation as removing it causes at least one call to fail.  Might need changes in the inner API.  Will look at it and get back asap.